worldedit-core/src/main/java/com/sk89q/worldedit/command/ClipboardCommands.java

```diff
@@ -66,6 +68,22 @@
 @CommandContainer(superTypes = CommandPermissionsConditionGenerator.Registration.class)
 public class ClipboardCommands {
 
+    /**
+     * Throws if the region would allocate a clipboard larger than the block change limit.
+     *
+     * @param region The region to check
+     * @param session The session
+     * @throws MaxChangedBlocksException if the volume exceeds the limit
+     */
+    private void checkRegionBounds(Region region, LocalSession session) throws MaxChangedBlocksException {
+        Region clipboardBounds = region instanceof CuboidRegion
```

Comment:
Can you make this a method on `Region`:
```java
default CuboidRegion getBoundingBox() {
    return new CuboidRegion(getMinimumPoint(), getMaximumPoint());
}
```
and just override in `CuboidRegion`? I feel like this is generically useful.